src/compas/geometry/_core/angles.py

```diff
@@ -352,6 +352,30 @@ def angles_points_xy(a, b, c, deg=False):
     v = subtract_vectors_xy(c, a)
     return angles_vectors_xy(u, v, deg)
 
+def angle_planes(a, b, deg=False):
+    r""" Compute the smallest angle between the two normal vectors of two planes.
```

Comment:
the raw string prefix is only necessary if you use latex syntax in the docstring...

Reply:
oh! i didn't know that. good to know, copied. thank you very much for the comment